### PR TITLE
Fix flake8 issues except line too long

### LINKED ISSUE #37: Misc fixes
There are a couple of things that the linter that i use ([Flake8](https://github.com/PyCQA/flake8)) complains about (i excluded the `line too long` ones), these are the ones may be worth fixing: 
- E703 statement ends with a semicolon (**dictionary.py**, line **40**)
- E722 do not use bare 'except' (**cms.py**, lines **43, 93, 124 and 172**)
- E713 test for membership should be 'not in' (**cms.py**, line **286**)

These ones are minor but may also worth fixing:
- W292 no newline at end of file (**print_utils.py**, line **54**)
- W391 blank line at end of file (**drupal.py**, line **40**) _(there are multiple blank lines actually)_
- W391 blank line at end of file (**joomla.py**, line **21**)
- W504 line break after binary operator (**main.py**, lines **52 and 137**)
- W504 line break after binary operator (**wordpress.py**, line **77**)
- E128 continuation line under-indented for visual indent (**main.py**, lines **53 and 138**)
- F841 local variable 'spinner' is assigned to but never used (**main.py**, line **120**)

_Note: W504 and E128 are related to each other_

PD: When solving W504, some flake8 versions could complain about W503, but seems like it's preferable that operators remain aligned.
- [Flake8's W504 explanation](https://www.flake8rules.com/rules/W504.html)
- [PEP8's "Should a Line Break Before or After a Binary Operator?"](https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator)

### PR DESCRIPTION
**Pull request by dukebody**
Resolves #37.

For W504 I tried to solve it not needing the binary operators or putting everything in the same line.

### CODE REVIEW

File: masc/main.py [hunk 1]
```diff
@@ -16,6 +14,8 @@
 from masc.dictionary import Dictionary
 from masc.masc_utils import MascUtils
 
+sys.path.extend(['.', '..'])
```
Comment (dukebody) [inline masc/main.py]: I'm worried about this being before all the `masc` imports for a reason, but IMHO it looks like not very good practice to extend the system path within the script - looks like issues with how the software is packaged and can even be dangerous.

However running masc locally after the change apparently works, so I'm not super concerned :)

Comment (ibLeDy) [reply masc/main.py]: It is there on the first place to be able to execute `python3 masc/main.py` or `python3 main.py` (it was the only way i managed to do it), as after the packaging refactor we could only execute the script (after cloning the repo, without installing the package) via `python3 -m masc.main`.

I see that it has been moved to after the masc imports, but i think it defeats the purpose of the statement being there in the first place, as it does not prevent the `ModuleNotFound` error anymore:

image

With that said, I agree with your statements, and i think the better option would be to get rid of it.

_If we don't, we may want to ignore those E402 warnings with a `# noqa: E402` comment._

Comment (dukebody) [reply masc/main.py]: I think using `sys.path.extend(['.', '..'])` is a hack and the recommended way to develop should be using `pip install -e .` from the root. I will do changes to make this work and update the docs.

Comment (dukebody) [reply masc/main.py]: @ibLeDy I have removed the path mangling and created https://github.com/sfaci/masc/issues/42 to update the recommended ways to install and develop masc.

**Comment by dukebody** [comment]
@sfaci can you take a look and tell me what you think?

File: masc/main.py (hunk 1 above)
Comment (ibLeDy) [reply masc/main.py]: Indeed, i agree.

**Review by ibLeDy — APPROVED** [review]

**Comment by dukebody** [comment]
@sfaci can we merge this? :D

**Comment by sfaci** [comment]
Done!

**Pull request merged**